securedrop/journalist_app/main.py

```diff
@@ -41,22 +45,24 @@ def login():
         return render_template("login.html")
 
     @view.route('/logout')
-    def logout():
+    def logout() -> werkzeug.Response:
         session.pop('uid', None)
         session.pop('expires', None)
         session.pop('nonce', None)
         return redirect(url_for('main.index'))
 
     @view.route('/org-logo')
-    def select_logo():
+    def select_logo() -> werkzeug.Response:
+        if current_app.static_folder is None:
```

Comment:
Same question as above.

Reply:
Same thing, it is typed as `Optional[str]`